jrpcfs/config.go

```diff
@@ -299,5 +309,16 @@ func Down() (err error) {
 	err = nil
 	jsonRpcServerDown()
 	ioServerDown()
+
+	globals.Lock()
+	for _, listner := range globals.listners {
+		listner.Close()
+	}
+
+	for elm := globals.connections.Front(); elm != nil; elm = elm.Next() {
+		conn := elm.Value.(net.Conn)
+		conn.Close()
+	}
+	globals.Unlock()
```

Comment:
And, finally, I expected to see a globals. connWaitGroup.Wait() call here, right?

Reply:
YES - fixed it.